ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/aaa.yml

```diff
@@ -60,7 +60,11 @@ aaa_authorization:
   serial_console: true
   commands:
     all_default: group aaaAuth
-
+    privilege:
+      - level: "10,15"
+        group: "tacacs+ local"
+      - level: "5"
+        group: "radius"
```

Comment:
Please add one empty line after line 67 for separation. Sorry, I missed many things.